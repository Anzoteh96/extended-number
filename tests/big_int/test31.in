-2862313613384199958578218558917499637153240285063051886309297609537191803856244581098165712046
820717205098533709982077762127360087251726856294823574268415897534097079671005248192017095564
