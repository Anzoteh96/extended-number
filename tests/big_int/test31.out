Enter two numbers n and m:
n is -2862313613384199958578218558917499637153240285063051886309297609537191803856244581098165712046
m is 820717205098533709982077762127360087251726856294823574268415897534097079671005248192017095564
n + m is -2041596408285666248596140796790139549901513428768228312040881712003094724185239332906148616482
n - m is -3683030818482733668560296321044859724404967141357875460577713507071288883527249829290182807610
n * m is -2349150028892165560624383121409289222039104650770777363640537683039735836551499200841117308355777461402590762697340943890508727707498283047124547653926182225416569037406451003044887963944
n / m is -3
n % m is -400161998088598828631985272535419375398059716178581163504049916934900564843228836522114425354
